### ISSUE TITLE
Game tabs out whenever i click on anything, impossible to play, cannot even access settings without game tabbing out 

### ISSUE THREAD

**Issue by lasdah** [Uuvana-Studios/longvinter-windows-client#1687]
Clicking anywhere in game causes the window to tab out, controller is unplugged game has be reinstalled before still same issue. I cannot change any In game setting as well because anything I click on causes tab out

**Comment by phloyddd** [comment]
I think maybe you posted this on reddit earlier, someone asked me about it. I'm not aware of any other players having ever reported this issue. I think the issue is likely to be an issue with your PC. You might read through this thread on reddit it has quite a few things you could try to fix it. https://www.reddit.com/r/techsupport/comments/175zz0w/games_randomly_alttab_out/

**Comment by lasdah** [comment]
> I think maybe you posted this on reddit earlier, someone asked me about it. I'm not aware of any other players having ever reported this issue. I think the issue is likely to be an issue with your PC. You might read through this thread on reddit it has quite a few things you could try to fix it. https://www.reddit.com/r/techsupport/comments/175zz0w/games_randomly_alttab_out/

is there a way to change the game setting without launching it? Because i want to try disabling full screen mode to see if it fix the issue, however i cant access the settting in game without the game tabbing out

**Comment by phloyddd** [comment]
Yeah took me a bit to figure this out but see if it works for you. You need to open a run command dialog box so hit Windows Key + R then type %AppData% then you navigate back a page so hit Appdata in the address bar at the top of the screen then navigate to Local\Longvinter\Saved\WindowsNoEditor and open Settings.ini in Notepad. For borderless fullscreen instead of regular fullscreen you want the last line of that file to say r.setres=1920x1080wf. Hopefully it does say f at the end and doesn't already say wf because that should let you know if that was the issue but i think the last chars should be w=windowed f=fullscreen wf=windowed fullscreen (borderless). You might wanna join the discord server too if you need more help, honestly this github page barely gets checked and never gets checked by staff and I'm just a helpful player lol https://discord.gg/longvinter

**Comment by lasdah** [comment]
Could you help me ask if i can be verified in the discord server? Because it requires phone number verification which i cannot do since i've been banned on discord previously and they do not allow reusing phone number.

**Comment by lasdah** [comment]
> Yeah took me a bit to figure this out but see if it works for you. You need to open a run command dialog box so hit Windows Key + R then type %AppData% then you navigate back a page so hit Appdata in the address bar at the top of the screen then navigate to Local\Longvinter\Saved\WindowsNoEditor and open Settings.ini in Notepad. For borderless fullscreen instead of regular fullscreen you want the last line of that file to say r.setres=1920x1080wf. Hopefully it does say f at the end and doesn't already say wf because that should let you know if that was the issue but i think the last chars should be w=windowed f=fullscreen wf=windowed fullscreen (borderless). You might wanna join the discord server too if you need more help, honestly this github page barely gets checked and never gets checked by staff and I'm just a helpful player lol https://discord.gg/longvinter

Borderless Fullscreen fixed the issue, no idea why fulscreen stopped working as it was doing fine previously

**Comment by phloyddd** [comment]
I'll try to pass the message along that you're unable to join the server because you can't get your account verified due to a phone number but I'm not sure there is any way for them to bypass this without completely disabling the requirement for verified accounts and allowing you to join and then enabling it again which honestly is a lot of work. Honestly, I'm sure not the solution you want but if you're able to get a new phone number with your carrier that might be the best solution. More and more discord servers are going to be requiring verified accounts considering how many scams run rampant on discord now and the community features of discord require it to be enabled.
